data/mods/Xedra_Evolved/mutations/paraclesians/homullus_mutations.json

```
@@ -167,6 +167,143 @@
       }
     ]
   },
+  {
+    "type": "mutation",
+    "id": "HOMULLUS_CRAFTING_BONUS_1",
+    "name": { "str": "The Tool-Maker" },
+    "points": 3,
+    "visibility": 0,
+    "ugliness": 0,
+    "description": "Though they had no claws or fangs or armored skin, humanity mastered the world through their tools.  When near civilization, your crafting speed is increased.",
+    "changes_to": [ "HOMULLUS_CRAFTING_BONUS_2" ],
+    "category": [ "HOMULLUS" ],
+    "enchantments": [
+      {
+        "condition": {
+          "or": [
+            { "u_near_om_location": "road_curved", "range": 3 },
+            { "u_near_om_location": "road_four_way", "range": 3 },
+            { "u_near_om_location": "road_tee", "range": 3 },
+            { "u_near_om_location": "road_straight", "range": 3 },
+            { "u_near_om_location": "road_end", "range": 3 },
+            { "u_near_om_location": "road_sw", "range": 3 },
+            { "u_near_om_location": "road_ne", "range": 3 },
+            { "u_near_om_location": "road_ew", "range": 3 },
+            { "u_near_om_location": "road_ns", "range": 3 },
+            { "u_near_om_location": "road_nesw", "range": 3 },
+            { "u_near_om_location": "road", "range": 3 },
+            { "u_near_om_location": "dirt_road", "range": 3 },
+            { "u_near_om_location": "subway", "range": 3 },
+            { "u_near_om_location": "FACTION_CAMP_ANY", "range": 6 },
+            { "u_near_om_location": "evac_center_18", "range": 6 },
+            { "u_near_om_location": "isolated_road_field_0", "range": 6 },
+            { "u_near_om_location": "lumbermill_0_0_ocu", "range": 6 },
+            { "u_near_om_location": "ranch_camp_41", "range": 6 },
+            { "u_near_om_location": "godco_5", "range": 6 },
+            { "u_near_om_location": "smallscrapyard_ocu", "range": 4 },
+            { "u_near_om_location": "robofachq_surface_entrance", "range": 6 }
+          ]
+        },
+        "values": [ { "value": "CRAFTING_SPEED_MULTIPLIER", "multiply": -0.15 } ]
```

`Positive values decrease crafting time, negative values increase it.`
```suggestion
        "values": [ { "value": "CRAFTING_SPEED_MULTIPLIER", "multiply": 0.15 } ]
```